cray_product_catalog/catalog_update.py

```diff
@@ -232,6 +246,15 @@ def update_config_map(data, name, namespace):
             else:
                 LOGGER.exception("Error calling replace_namespaced_config_map")
 
+def split_catalog_data(data, main_cm_data, sub_cm_data):
```

Comment:
@lathanm instead of updating as reference, return the data, its confusing in python language. Folks with non python background will become confused.